osu.Game/Overlays/Comments/DrawableComment.cs

```diff
@@ -298,7 +335,12 @@ public RepliesButton(int count)
 
             protected override void OnExpandedChanged(ValueChangedEvent<bool> expanded)
             {
-                text.Text = $@"{(expanded.NewValue ? "[+]" : "[-]")} replies ({count})";
+                text.Text = $@"{(expanded.NewValue ? "[+]" : "[-]")} replies ({repliesCount})";
+            }
+
+            protected override void OnChildrenChanged(ValueChangedEvent<List<Comment>> children)
+            {
+                Alpha = (children.NewValue.Count == 0 && repliesCount > 0) || repliesCount == 0 ? 0 : 1;
```

Comment:
First off, this check can be simplified to
```csharp
repliesCount == 0 || children.NewValue.Count == 0
```
but mostly I'm confused with what this does. Does `children` contain only the comments that are going to be *added* below (therefore meaning that there's nothing more to load, so the button can be hidden)? I don't see any other reason for hiding the button if it's empty; if so, mind changing the parameter naming to reflect that?

Reply:
Resolved by latest refactoring